lakesoul-flink/src/main/java/org/apache/flink/lakesoul/source/LakeSoulOneSplitRecordsReader.java

```diff
@@ -129,7 +133,11 @@ private void initializeReader() throws IOException {
         }
 
         if (filter != null) {
```

Comment:
Will two kinds of filter cause conflict?

Reply:
No, `filterSter` is null forever in above code. It is used to debug the difference of result datafusion::expr of two kinds filters.